### PR TITLE
Fixed Issue #442

### PR DESCRIPTION
**Pull request by girishkumarkh**
Added footer to issue #442

### CODE REVIEW

File: djangoproject/scss/_style.scss [hunk 1]
```diff
@@ -1022,13 +1022,21 @@ blockquote {
 					background: url(../img/logo-rackspace.svg) no-repeat left center;
 				}
 			}
+
+			&.elastic {
+				background: url(../img/logo-elastic.png) no-repeat left center;
+				.mdzr-svg & {
+					background: url(../img/logo-elastic.svg) no-repeat left center;
+					background-size: contain;
```
Comment (bmispelon) [inline djangoproject/scss/_style.scss]: What does `background-size: contain` do?

Comment (girishkumarkh) [inline djangoproject/scss/_style.scss]: Basically the elastic search image is little bigger than the background container, by adding `background-size: contain;` would align them within the container.

**Comment by bmispelon** [comment]
Squashed it down to one commit and merged in c3faec63db8e16e6d69b4f531be4bd66e8e2d4e6.

Thanks! :rocket:

**Comment by girishkumarkh** [comment]
👍

**Pull request closed without merging**